<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rubiks Cube Solver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="cubeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="5,5,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="currCubeText"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="currCubeLabel">
       <property name="text">
        <string>Current Cube State</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="scrambleOutput"/>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="inputCubeText"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="inputStringLabel">
       <property name="text">
        <string>Input String</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="scrambleLabel">
       <property name="text">
        <string>Scramble Moves</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="solutionLabel">
       <property name="text">
        <string>Solution</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextBrowser" name="solutionOutput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="inputDoneBtn">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>INPUT DONE</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="scrambleBtn">
       <property name="text">
        <string>SCRAMBLE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="solveBtn">
       <property name="text">
        <string>SOLVE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>168</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="turnBtns">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="BBtn">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="UBtn">
       <property name="text">
        <string>U</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="DprimeBtn">
       <property name="text">
        <string>D'</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="BprimeBtn">
       <property name="text">
        <string>B'</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="RBtn">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="FBtn">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="LBtn">
       <property name="text">
        <string>L</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="FprimeBtn">
       <property name="text">
        <string>F'</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="UprimeBtn">
       <property name="text">
        <string>U'</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="DBtn">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="LprimeBtn">
       <property name="text">
        <string>L'</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="RprimeBtn">
       <property name="text">
        <string>R'</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
